plugins/orchestrator/src/components/WorkflownstancesViewerPage/AssessmentResultViewer.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from 'react';
+
+import { InfoCard } from '@backstage/core-components';
+
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Chip,
+  Link,
+  Typography,
+} from '@material-ui/core';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+
+interface AssessmentResultViewerProps {
+  result: Record<string, unknown> | string | undefined;
+}
+
+interface WorkflowOption {
+  id: string;
+  name: string;
+}
+
+export const AssessmentResultViewer = (props: AssessmentResultViewerProps) => {
+  const { result } = props;
+
+  const jsonSource = useMemo(() => {
+    if (!result) {
+      return undefined;
+    }
+    if (typeof result === 'string') {
+      return JSON.parse(result);
+    }
+    return result;
+  }, [result]);
+
+  const idToUrl = (id: string) => {
+    return `/orchestrator/workflows/${id}/execute`;
+  };
+
+  const keyToTitle = (key: string) => {
+    const title = key.replace(/([a-z])([A-Z])/g, '$1 $2');
+    return title.charAt(0).toUpperCase() + title.slice(1);
+  };
+
+  const accordionProps = (items: WorkflowOption | WorkflowOption[]) => {
+    if (!Array.isArray(items)) return { expanded: true };
+    if (Array.isArray(items) && items.length === 0) return { disabled: true };
+    return {};
+  };
+
+  const workflowLinks = (items: WorkflowOption | WorkflowOption[]) => {
+    if (!Array.isArray(items))
+      return (
+        <>
+          <Link href={idToUrl((items as WorkflowOption).id)}>
+            {(items as WorkflowOption).name}
+          </Link>
+          &nbsp;&nbsp;
+          <Chip label="Recommended" size="small" />
+        </>
+      );
+    return items.map(item => {
+      const workflowOption: WorkflowOption = item;
+      return (
+        <>
```

Comment:
I havn't tried it but isn't React complaining about missing `key` property? (in the browser console)

Reply:
@mareklibra it's not. The key has been added in the Accordion tag that wrapped all link items.